### ISSUE TITLE
Description not accurate

### ISSUE THREAD

**Issue by naseemhaq** [x-j/ps1scriptify#5]
Description in GitHub About section should say "tool to create powershell script to run a python script"

**Comment by mmcleish-mb** [comment]
i agree.  i thought it was converting your PY code to a PS1 code.  but no, this is just a wrapper to execute a PY within PS.
